http_client.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+var client = new(http.Client)
+
+func checkAPIErr(err error) {
+	if err != nil {
+		log.Fatalf("API error: %v. Please contact Rainforest support.", err)
+	}
+}
+
+func addAuthHeaders(req *http.Request) {
+	req.Header.Add("CLIENT_TOKEN", apiToken)
+	req.Header.Add("Content-Type", "application/json")
+}
+
+func postRequest(url string, body []byte) []byte {
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(body))
+	checkAPIErr(err)
```

Comment:
FWIW I would panic here because if this error exists, the code is wrong, not the client or the API.